plugins/lsp/language-servers/default.nix

```
@@ -248,6 +248,13 @@ let
       ];
       settings = cfg: { json = cfg; };
     }
+    {
+      name = "jsonnet";
```

```suggestion
      name = "jsonnet-ls";
```
To be consistent with the `nvim-lspconfig` naming.

---

Done